src/link/internal.tsx

```diff
@@ -121,20 +122,29 @@ const InternalLink = React.forwardRef(
       fireCancelableEvent(onFollow, { href, external, target: anchorTarget }, event);
     };
 
+    const fireClickEvent = (event: React.MouseEvent | React.KeyboardEvent) => {
+      const { altKey, ctrlKey, metaKey, shiftKey } = event;
+      const button = 'button' in event ? event.button : 0;
+      fireNonCancelableEvent(onClick, { altKey, button, ctrlKey, metaKey, shiftKey });
```

Comment:
can we add a comment here why this is nonCancelable, so that it doesn't get "fixed" in the future?